snuba/web/views.py

```diff
@@ -97,10 +98,12 @@ def check_clickhouse() -> bool:
 application.url_map.converters["dataset"] = DatasetConverter
 
 
-@application.errorhandler(BadRequest)
-def handle_bad_request(exception: BadRequest):
+@application.errorhandler(InvalidJsonRequestException)
+def handle_invalid_json(
+    exception: InvalidJsonRequestException,
+) -> Tuple[str, int, Mapping[str, str]]:
     cause = getattr(exception, "__cause__", None)
-    if isinstance(cause, json.errors.JSONDecodeError):
```

Comment:
I am not entirely sure why this was working.

Reply:
```
>>> json.JSONDecodeError
<class 'simplejson.errors.JSONDecodeError'>
```